README.md

```diff
@@ -1,5 +1,71 @@
 Lucene JBoss Module
 ===================
 
+Packaging of [Apache Lucene](http://lucene.apache.org) as a [JBoss Module](https://docs.jboss.org/author/display/MODULES/Home).
+
+Historically the Hibernate Search project has been releasing such modules, for convenience of Apache Lucene
+users running applications on WildFly or JBoss EAP.
+
+We now moved these artifacts to a "Lucene only" bundle so that other projects using Lucene don't have to
+download the Hibernate Search specific modules, and to attempt to have a single consistent distribution
+of such modules.
+
+This should also make it easier to release a new bundle of these modules as soon as a new Lucene version
+is released, without necessarily waiting for Hibernate Search to have adopted the new Lucene version.
+
+== Versions
+
+While it would be tempting to follow the versions of the bundled Apache Lucene versions,
+this wouldn't allow to release improvements nor bugfixes in the module structure itself.
+
+We will use an X.Y.Z pattern with an optional qualifier as defined on [JBoss Project Versioning](https://developer.jboss.org/wiki/JBossProjectVersioning), where X.Y will match the major and minor versions of the included Lucene version.
+
+The micro version will not necessarily match the Lucene version: we will increase this for any change;
```

Comment:
That's a great idea. I didn't want to abuse the "qualifier" section as prod team uses that section to distiguish from community releases, but I see that the "JBoss Project Versioning" I linked to actually allow this. +1